electron/main.js

```diff
@@ -322,7 +335,7 @@ const createMainWindow = async () => {
   // Get the agent's current state
   ipcMain.handle('health-check', async (_event) => {
     try {
-      const response = await fetch('http://127.0.0.1:8716/healthcheck', {
+      const response = await fetch('https://localhost:8716/healthcheck', {
```

Comment:
we open each agent UI on `http://127.0.0.1:8716` and also communicate inside through http, do you think it all should be changed? @OjusWiZard

Reply:
Oh good catch! I kept the agent UI URLs the same (`127.0.0.1`) and only changed the backend URLs to `localhost`, I probably missed this one.